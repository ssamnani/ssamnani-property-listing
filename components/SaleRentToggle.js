import { Radio } from "antd";

function SaleRentToggle() {
  return (
    <>
      <Radio.Group defaultValue="sale" buttonStyle="solid" size="large">
        <Radio.Button value="sale">Sale</Radio.Button>
        <Radio.Button value="rent">Rent</Radio.Button>
      </Radio.Group>
    </>
  );
}

export default SaleRentToggle;
