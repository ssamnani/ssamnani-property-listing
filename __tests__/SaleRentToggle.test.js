import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import SaleRentToggle from "../components/SaleRentToggle";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render("<SaleRentToggle />", div);
});

// Snapshot Testing
it("matches snapshot", () => {
  const tree = renderer.create(<SaleRentToggle />).toJSON();
  expect(tree).toMatchSnapshot();
});
