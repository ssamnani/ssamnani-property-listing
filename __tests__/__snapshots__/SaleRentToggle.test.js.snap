// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="ant-radio-group ant-radio-group-solid ant-radio-group-large"
>
  <label
    className="ant-radio-button-wrapper ant-radio-button-wrapper-checked"
  >
    <span
      className="ant-radio-button ant-radio-button-checked"
      style={Object {}}
    >
      <input
        checked={true}
        className="ant-radio-button-input"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value="sale"
      />
      <span
        className="ant-radio-button-inner"
      />
    </span>
    <span>
      Sale
    </span>
  </label>
  <label
    className="ant-radio-button-wrapper"
  >
    <span
      className="ant-radio-button"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-radio-button-input"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value="rent"
      />
      <span
        className="ant-radio-button-inner"
      />
    </span>
    <span>
      Rent
    </span>
  </label>
</div>
`;
